src/pages/api/newsletter/subscribe.js

```diff
@@ -0,0 +1,23 @@
+export default async function handler(req, res) {
+  if (req.method === 'POST') {
+    await fetch(`https://api.hubapi.com/contacts/v1/contact/?hapikey=${process.env.HUBSPOT_API_KEY}`, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      redirect: 'follow',
+      referrerPolicy: 'no-referrer',
+      body: JSON.stringify({
+        properties: [{ property: 'email', value: req.body.email }],
+      }),
+    })
+      .then((response) => {
+        res.status(response.status).send(response.json());
```

Comment:
I am not sure what the response is, but I guess that we may just want to return true or false and not the whole subscribe object